lsp4clj/deps.edn

```diff
@@ -0,0 +1,10 @@
+{:deps {com.taoensso/timbre {:mvn/version "5.1.2"}
+        org.eclipse.xtend/org.eclipse.xtend.lib {:mvn/version "2.25.0"}
+        org.eclipse.lsp4j/org.eclipse.lsp4j {:mvn/version "0.12.0"  :exclusions [org.eclipse.xtend/org.eclipse.xtend.lib
+                                                                                 com.google.code.gson/gson]}
+        org.clojure/data.json {:mvn/version "2.4.0"}
+        com.google.code.gson/gson {:mvn/version "2.9.0"}
```

Comment:
Shouldn't we add `org.clojure/clojure` as well? I never know if I should add org.clojure/clojure or not on deps.edn, I think it's important to know which clojure rely when building jar/uberjar

Reply:
Yeah, I don't know either, let's be safe though 😄